src/KafkaFlow.Abstractions/MessageEventContext.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace KafkaFlow
+{
+    public class MessageEventContext
+    {
+        public IMessageContext MessageContext { get; set; }
+
+        public IDependencyResolver DependencyResolver { get; set; }
```

Comment:
no setters and initialization from ctor. Or just use a record, now we can :)

Reply:
I tried to use a record but my VS kept saying that I couldn't use C#9 features in .netstandard2.0 projects 🤔

Reply:
Done, please review